semantic-analysis/src/Analysis/Concrete.hs

```diff
@@ -180,9 +174,5 @@ instance ( Has (A.Env A.PAddr) sig m
       _             -> fail "expected Bool"
     L (DString s) -> pure (String s <$ ctx)
     L (DDie msg)  -> fail (show (quote msg))
-    L (DLet n v b) -> do
-      addr <- A.alloc n
-      addr A..= v
-      A.bind n addr $ hdl (b v <$ ctx)
```

Comment:
`DLet` ended up being unused since it's macro-expressible.

Reply:
Do you implement this anywhere?

Reply:
Yep! In the `Analysis.Syntax` module, which is (not at all coincidentally) the only place it's used.